Source/Scene/Model.js

```diff
@@ -4094,20 +4018,32 @@ define([
                 }
 
                 this._loadResources = new LoadResources();
-                parse(this);
             }
+            parse(this);
         }
 
         var loadResources = this._loadResources;
         var incrementallyLoadTextures = this._incrementallyLoadTextures;
         var justLoaded = false;
 
         if (this._state === ModelState.LOADING) {
-            // Create WebGL resources as buffers/shaders/textures are downloaded
-            createResources(this, frameState);
-
             // Transition from LOADING -> LOADED once resources are downloaded and created.
             // Textures may continue to stream in while in the LOADED state.
+            if (loadResources.pendingBufferLoads === 0 && loadResources.pendingShaderLoads === 0) {
+                if (!this._updatedGltfVersion) {
+                    updateVersion(this.gltf);
```

Comment:
I'm confused about the order of operations:
* Should this `updateVersion` be one of the first things that happens?  Then `addDefaults` and `processModelMaterialsCommon`, for example, can operate on glTF 1.1?  This will, of course, require more gltf-pipeline changes, but we want to natively support just glTF 1.1 and have a 1.0 -> 1.1 conversion step early in the pipeline.  This will be the simplest organization.
* I'll probably figure this out as I look at more code, but why is `addPipelineExtras` required?  Because the other gltf-pipeline stages require it, right?  Why are we also using `extras` for loading resources here?  It seems like we should treat this as a gltf-pipeline implementation detail and not use it for multiple things.  Are the `extras` removed later?